core/tests/webdriverio_utils/AdminPage.js

```diff
@@ -151,8 +151,25 @@ var AdminPage = function() {
     return null;
   };
 
-  // Remove this method after the end_chapter_celebration feature flag
+  // Remove this method after the checkpoint_celebration feature flag
   // is deprecated.
+  this.getCheckpointCelebrationFeatureElement = async function() {
+    var featureFlagElements = await featureFlagElementsSelector();
+    var count = featureFlagElements.length;
+    for (let i = 0; i < count; i++) {
+      var elem = featureFlagElements[i];
+      if ((await elem.$(featureNameLocator).getText()) ===
+          'checkpoint_celebration') {
+        return elem;
+      }
+    }
+
+    return null;
+  };
+
+  // This function is meant to be used to enable a feature gated behind
+  // a feature flag in prod mode, which is the server environment the E2E
+  // tests are run in.
```

Comment:
The function mentioned in this comment is one I added as part of M1 of the project. The need was to have a function that can flip a feature flag to 'ON' to allow for E2E tests to test gated features. This would allow for the feature to be tested without having to remove the feature flag.

Back then, I assumed this was a one-time thing, but I now realize that this function could be used for any gated feature in general, hence the comment change.